x-pack/plugins/apm/public/context/url_params_context/helpers.test.ts

```diff
@@ -52,6 +30,43 @@ describe('url_params_context helpers', () => {
       });
     });
 
+    describe('when rangeFrom or rangeTo are falsy', () => {
+      it('returns the previous state', () => {
+        jest.spyOn(console, 'warn').mockImplementationOnce(() => {});
```

Comment:
Perhaps use the existing helper `disableConsoleWarning(...)`

Reply:
The purpose of that was more to disable all warnings that match a certain pattern. In this case we get a warning about the date parsing function not getting a valid date. Since we're testing what it does with an invalid date we don't want to log the message here but we would want to log it normally.

Reply:
Makes sense